system/src/documents/ActorSD.mjs

```diff
@@ -525,6 +525,17 @@ export default class ActorSD extends Actor {
 			data.itemDamageBonus = item.system.bonuses.damageBonus * damageMultiplier;
 		}
 
+		/* Attach Special Ability if part of the attack.
+			Created in `data.itemSpecial` field.
+			Can be used in the rendering template or further automation.
+		*/
+		if (item.system.damage.special) {
+			let itemSpecial = data.actor.items.find(e => e.name === item.system.damage.special);
```

Comment:
Are you using linting? This should be picked up by the linting config to use `const` instead. Check out the wiki for what we recommend using!
https://github.com/Muttley/foundryvtt-shadowdark/wiki

Reply:
I am in fact using ESLINT as described.

Screenshot of what happens if I change to `var` instead
![image](https://github.com/Muttley/foundryvtt-shadowdark/assets/319590/1d835909-cb60-4535-a732-630e1ca8d478)

But ESLint gives me a clean bill of health with either `let` or `const`.
![image](https://github.com/Muttley/foundryvtt-shadowdark/assets/319590/03c7b662-589a-4f4c-9484-3ced4a8694b6)
![image](https://github.com/Muttley/foundryvtt-shadowdark/assets/319590/9b1daaaf-b8d6-4b5f-9002-0e5263f7b8a0)

The tool is clearly on. It's already caught some "long lines of code" issues elsewhere. It's catching the `var` exception. So this is some type of corner case where the linter seems to be giving me a pass?

If you prefer `const`, you should be able to edit and merge. I'm really good either way.

Reply:
Huh, that is strange. Well, I do prefer `const` everywhere possible, and if you're doing any changes to the comment below, could you change this while on it?

Reply:
Changes committed directly.

Reply:
> If you prefer `const`, you should be able to edit and merge. I'm really good either way.

Yeah, always use `const` where possible or `let` if you need to change the value.

Never use `var` due to hoisting; it's considered a code smell these days (and rightly so).